radixdlt-core/radixdlt/src/main/java/com/radixdlt/NetworkGenesis.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * (C) Copyright 2021 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package com.radixdlt;
+
+import com.radixdlt.atom.Txn;
+import com.radixdlt.networks.Network;
+
+import java.util.Optional;
+
+public final class NetworkGenesis {
+	private NetworkGenesis() { }
+
+	//Moved from Network
+	public static Optional<Txn> txn(Network network) {
```

Comment:
There should be no layer of indirection between Network and hardcoded genesis, so Genesis Txn should be in Network class.

Reply:
If this is a problem because radixdlt-common doesn't have `Txn` then we can just change the type to be a Hex string.

Reply:
Yep, the reason was that I did need the `Txn` in the `radixdlt-java-common` to keep this method inside `Network`. 
Moved it back to `Network` and changed return type to hex string.